proposals/oras.md

```diff
@@ -0,0 +1,177 @@
+	rocketFileName := "rocket.txt"
+	rocketMediaType := "text/plain"
+	rocketContent := []byte("🚀")
+	rocketDescriptor := store.Add(rocketFileName, rocketMediaType, rocketContent)
+
+	rocketConfigMediaType := "application/vnd.acme.rocket.config.v1+json"
+	rocketConfigContent := []byte(fmt.Sprintf("{\"RocketVersion\":\"%s\"}", rocketVersion))
+	rocketConfigDescriptor := store.Add("", rocketConfigMediaType, rocketConfigContent)
+
+	ref := fmt.Sprintf("%s/%s:%s", registryRootURL, registryNamespace, rocketVersion)
+	_, err := oras.Push(ctx, resolver, ref, store, []ocispec.Descriptor{rocketDescriptor},
+		oras.WithConfig(rocketConfigDescriptor))
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println("Pushed to", ref)
+	fmt.Printf("\nTry:\n\ncurl -s -H 'Accept: application/vnd.oci.image.manifest.v1+json' \\\n" +
+		"    %s/v2/%s/manifests/%s | jq\n", registryRootURL, registryNamespace, rocketVersion)
+}
+```
+
+You can see all features in the project [README](https://github.com/deislabs/oras/blob/master/README.md).
+
+#### Adoption ####
+
+The following projects are already successfully using ORAS to work with custom artifacts:
+
+- [Helm](https://github.com/helm/helm/search?q=oras)
+- [Conftest](https://github.com/instrumenta/conftest/search?q=oras)
+- [Singularity](https://github.com/sylabs/singularity/search?q=oras)
+
+
+
+## Proposal ##
+Change the ownership of the existing umoci project from deislabs:
+
+  https://github.com/deislabs/oras
+
+And move it inside the `opencontainers` organization:
+
+  https://github.com/opencontainers/oras
+
+The import paths will correspondingly be "github.com/opencontainers/oras" (oras does have some Go API users, but since the project will be renamed -- and GitHub will add a redirect -- there will be no significant downstream impact of the change).
+
+### Initial Maintainers ###
+Initial maintainers of the ORAS project would be:
+
+* Josh Dolitsky <josh@bloodorange.io> (@jdolitsky)
+* Shiwei Zhang <shizh@microsoft.com> (@shizhMSFT)
+* Sajay Antony <sajaya@microsoft.com> (@sajayantony)
+* Steve Lasker <steve.lasker@microsoft.com> (@stevelasker)
+* Jimmy Zelinskie <jimmyzelinskie@gmail.com> (@jzelinskie)
+* Matt Fisher <matt.fisher@microsoft.com> (@bacongobbler)
+
+### Code of Conduct ###
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md
+
+### Governance and Releases ###
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+It should be noted that since ORAS is not a specification, it is not bound by the ordinary quorum and voting rules for specification release.
+As such, new versions will be released as regularly as needed without the need for a quorum vote.
+
+Until there are enough additional maintainers, PRs will be merged with only one maintainer LGTM required.
```

Comment:
This doesn't quite appear to match the maintainers list (which has a fair number of people) -- two LGTMs shouldn't be an unreasonable requirement for PRs.

Reply:
2  ++